apollo-portal/src/main/config/application-oidc-sample.yml

```diff
@@ -13,6 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+server:
+  # apollo-portal 开启 https 时需要配置该项
+  forward-headers-strategy: framework
```

Comment:
What will happen if no https is enabled but server.forward-headers-strategy is set to framework? Or is it better to comment this config by default?

Reply:
`server.forward-headers-strategy=framework` is used to resolve the `x-forwarded-*` header, it can works well without https